server/handlers/postSurveys.test.js

```diff
@@ -0,0 +1,86 @@
+const request = require('supertest');
+const { initDb, closeDb } = require('../databaseConnection');
+const { ObjectID } = require('mongodb');
+
+beforeEach(() => {
+  return initDb();
+});
+
+afterEach(() => {
+  return closeDb();
+});
+
+describe('Testing POST to the /surveys endpoint', () => {
```

Comment:
Do error cases need to be tested when json is not okay

Reply:
This has been added.